_templates/component_v4/with-prompt/component.ejs.t

```diff
@@ -20,4 +21,6 @@ class <%= componentName %> extends Component< <% if(generateParts.indexOf('Props
   }
 }
 
+<%= componentName %>.register();
+
 export default <%= componentName %>;
```

Comment:
Since this is a breaking, can we start doing `export class X` instead?
Same for `/with-prompt/one-file.ejs.t`